incubator/hnc/internal/pkg/selector/selector.go

```diff
@@ -0,0 +1,67 @@
+package selector
+
+import (
+	"strings"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+
+	api "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha2"
+)
+
+func GetSelectorAnnotation(inst *unstructured.Unstructured) (string, bool) {
```

Comment:
Seems this func doesn't need export either, if we export `GetSelector` and `GetTreeSelector`?

Reply:
This is to check if a selector annotation exists. Maybe I should rename this function